### DISCUSSION TITLE
Mega Support

### DISCUSSION THREAD

**Discussion by xd003** [arghyac35/aria-telegram-mirror-bot#169] [Ideas]
Would be great to have mega to drive support . Can use this - https://github.com/jaskaranSM/megasdkrest

**Comment by arghyac35** [comment]
Okay I will look.. the repo readme is not very clear on how to setup, so if u have setup that repo then can say how you did that?

**Reply by xd003** [reply]
Ah i personally haven't used that repo but many bot developers used it to add mega support in their respective bots. I can send you link of such commits but all others are python based bots
